ecommerce/Model/Order_detail.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Order_detail extends Model
+{
+    protected $fillable = [
+        'product_id',
+        'order_id',
+        'number'
+    ];
+
+    public function product(){
```

Comment:
```suggestion
    public function products(){
```
Sửa hết các file tương tự